src/server/transforms.js

```diff
@@ -71,6 +71,9 @@ function buildErrorData(item, options, callback) {
   item.data.body = item.data.body || {};
   item.data.body.trace_chain = chain;
 
+  // clear out any existing message
+  delete item.data.body.message
```

Comment:
I'd like to not lose this information. I think the preferred way to handle this is to implement what the browser transforms do:

From https://github.com/rollbar/rollbar.js/blob/master/src/browser/transforms.js#L99
```js
function addBody(item, options, callback) {
  if (item.stackInfo) {
    addBodyTrace(item, options, callback);
  } else {
    addBodyMessage(item, options, callback);
  }
}
```

I would refactor the server-side to remove the `addMessageData()` transform and add the `addBody()` transform in its place. That code *may* need to be tweaked to remove the item.message in the case we have an error as well. In this case, we should set the `error.description = message`. @rokob can provide more details.